webpack.dev.js

```diff
@@ -0,0 +1,15 @@
+const merge = require('webpack-merge');
+const config = require('./webpack.config.js');
+const WriteFilePlugin = require('write-file-webpack-plugin');
+
+module.exports = merge(config, {
+  mode: 'development',
+  devtool: 'source-map',
+  plugins: [
+    ...config.plugins,
```

Comment:
Maybe I'm wrong but I believe you don't need to spread anything when using webpack-merge

Reply:
You're correct :wink: